plugin-energy-estimation-tdp/README.md

```diff
@@ -0,0 +1,75 @@
+# energy estimation tdp plugin
+
+## Table of Contents <!-- omit in toc -->
+
+- [introduction](#Introduction)
+- [Energy estimation tdp plugin](#Energy-estimation-tdp-plugin)
+  - [How to use](#how-to-use)
+  - [Prepare your environment](#prepare-your-environment)
+  - [Configuration](#Configuration)
+
+## Introduction
+
+This plugin estimate the energy consumption for pods based on the TDP value of the machine where pods are running.
+The TDP (Thermal Design Power) is the maximum amount of heat generated by a computer compnent. 
+Refer to https://en.wikipedia.org/wiki/Thermal_design_power for more details.
+
+In the first version of the plugin, we consider only the TDP of the CPU.
+
+ This plugin requires the cgroupv2 input plugin (k8s) as it needs the input measurements of cgroupsv2.
+
+ The estimation calculation is done using the following formula:
+
+ energy estimation = cgroupv2_cpu_total_usage * nb_vcpu/nb_cpu * tdp / 1000 000 / polling_interval
+
+cgroupv2_cpu_total_usage:   total usage of CPU in micro seconds for a pod 
+nb_vcpu:                    number of virtual CPU of the hosting machine where pod is running
+nb_cpu:                     number of physical CPU of the hosting machine where pod is running
+polling_interval:           polling interval of cgroupv2 input plugin
+
+
+## Energy estimation tdp plugin
+
+### How to use
+
+Just compile the app-agent of the alumet's github repository.
+
+```bash
+cargo run
+```
+
+
+The binary created by the compilation will be found under the target repository.
+
+### Prepare your environment
+
+To work this plugin needs k8s plugin configured, so the needed things are related to k8s plugin requirements:
+
+1. cgroup v2
+2. kubectl
+3. alumet-reader user
+
+### Configuration 
```

Comment:
```suggestion
### Configuration
```

Reply:
I do not understand how did you find the space at end line ? is there a trick ?

Reply:
You need to have the following extensions installed in your vscode: 

[markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
and
[Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one)